operators/src/util/gdal.rs

```diff
@@ -73,6 +73,17 @@ pub fn add_ndvi_dataset(ctx: &mut MockExecutionContext) -> DatasetId {
     id
 }
 
+// TODO: move test helper somewhere else?
+pub fn hide_gdal_errors_in_test() {
+    #[cfg(not(target_os = "windows"))]
+    let null_file = "/dev/null";
+
+    #[cfg(target_os = "windows")]
+    let null_file = "NUL";
+
+    gdal::config::set_config_option("CPL_LOG", null_file).unwrap();
```

Comment:
When setting `CPL_LOG` all messages will get routed into the nirvana using logic thats inside Gdal. This means that errors won't reach Rust code.
On the other hand, using `set_error_handler` Gdal messages need to get marshaled from Gdal to Rust so that they can be handled by the empty lambda.
All in all, the end result is the same: No Gdal messages will be printed to the console. Setting the config var would make the tests a tiny bit faster than using `set_error_handler`.

I see that it's unnecessary complicated. I will rewrite that part.

Reply:
0d4e7c1